Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -37,10 +36,12 @@ protected function configure()
         $this
             ->setName('doctrine:fixtures:load')
             ->setDescription('Load data fixtures to your database')
-            ->addOption('append', null, InputOption::VALUE_NONE, 'Append the data fixtures instead of deleting all data from the database first.')
+            ->addOption('append', null, InputOption::VALUE_NONE,
+                'Append the data fixtures instead of deleting all data from the database first.')
```

Comment:
This is a violation of PSR2, either have one argument par line or put them all on a single line, same below.